test/cli/test_cli.py

```diff
@@ -114,6 +117,45 @@ def test_should_walk_through_all_workflows_inside_package_tree(self):
         self.assertEqual('ID_2', res[1].workflow_id)
         self.assertNotEqual('@once', res[1].schedule_interval)
 
+    def test_walk_workflows_in_packages_should_not_walk_when_no_packages_given(self):
+        # when
+        res = walk_workflows_in_packages(TEST_PROJECT_DIR, [])
+
+        # then
+        self.assertEmpty(res)
+
+    def test_walk_workflows_in_packages_should_walk_through_all_workflows_without_transformer(self):
+        # when
+        res = walk_workflows_in_packages(TEST_PROJECT_DIR, ['test_module'])
+
+        # then
+        self.assertEqual([('ID_1', '@once'), ('ID_2', '@daily'), ('ID_3', '@daily'), ('ID_4', '@daily'), ('ID_5', '@daily')],
+            sorted(map(lambda w: (w.workflow_id, w.schedule_interval), res)))
+
+    def test_walk_workflows_in_packages_should_walk_through_all_workflows_inside_single_package(self):
```

Comment:
I'm pretty sure that "package walking" is already covered in our unit tests (like `test_should_walk_through_all_module_paths_inside_package_tree`). So maybe update existing tests to use new function (as it heavily depends on existing functions). 
Or at least reuse exiting "tests modules" (aka `Unused1.py` etc)

Reply:
Reused wherever possible, added just another package with correct modules and a special package with broken modules.